services/sensu/service.go

```diff
@@ -133,9 +136,10 @@ func (s *Service) prepareData(name, output string, level alert.Level) (*net.TCPA
 
 	postData := make(map[string]interface{})
 	postData["name"] = name
-	postData["source"] = c.Source
+	postData["source"] = hostname
```

Comment:
Would `source` be a better name for this option instead of `hostname`. While most of the time the source is a hostname the use of the name hostname initially confused me because Kapacitor can only have a single hostname and it doesn't change. But this is not talking about Kapacitor's hostname but rather the hostname of the source.